usagereportsv4/usage_reports_v4_integration_test.go

```diff
@@ -355,4 +373,108 @@ var _ = Describe(`UsageReportsV4 Integration Tests`, func() {
 			Expect(results).ToNot(BeEmpty())
 		})
 	})
+
+	Describe(`CreateReportsSnapshotConfig - Setup the snapshot configuration`, func() {
+		BeforeEach(func() {
+			shouldSkipTest()
+		})
+		It(`CreateReportsSnapshotConfig(createReportsSnapshotConfigOptions *CreateReportsSnapshotConfigOptions)`, func() {
+			fmt.Printf("TEST VALUES: %s", cosBucket)
+			fmt.Printf("TEST VALUES:%s", cosLocation)
```

Comment:
```suggestion
			fmt.Fprintf(GinkgoWriter, "COS Bucket: %s", cosBucket)
			fmt.Fprintf(GinkgoWriter, "COS Location:%s", cosLocation)
```
Please use `fmt.Fprintf(GinkgoWriter,...)` instead of `fmt.Printf(...)`.  This will ensure that output is displayed only when the `-ginkgo.v` command line option is used.
You can find other instances of this in the same source file.

Reply:
Oh this was a printf for testing. Will remove it.